### DISCUSSION TITLE
小爱同学pro增加 智能音箱 播放文本和  执行文本指令 功能。

### DISCUSSION THREAD

**Discussion by dldui-captain** [XiaoMi/ha_xiaomi_home#132] [Ideas]
现有小爱同学pro无法实现 智能音箱 播放文本和  执行文本指令 功能。希望增加一下，谢谢。
![image](https://github.com/user-attachments/assets/0d20ff40-eecf-4c1f-8888-cd50313c8614)

**Comment by zwlhome** [comment]
打开action调试

**Reply by williamcanon1** [reply]
Very usefully.

**Comment by SusanPhevos** [comment]
#51 是用HA自动化播放文本的方法。打开Action调试后，会多生成文本框，可以手动输入文本调试。

**Reply by qq173972819** [reply]
无法执行动作“text/set_value” 。 action exec failed, 智能音箱 播放文本(text.xiaomi_cn_604357173_lx06_play_text_a_5_1), invalid action params, 123

**Reply by pcr** [reply]
可以看一下他们的说明，参数要用 ["xxxxx"] 这样的格式提交
